nncf/tensorflow/sparsity/base_algorithm.py

```diff
@@ -36,13 +35,6 @@
     layer_metatypes.TFDenseLayerMetatype
 ]
 
-SPARSITY_TF_OP_METATYPES = [
```

Comment:
IMHO: These changes don't relative to the Common quantizer clasess PR.

Reply:
Indeed, but I tend to fix stuff whenever I see it. In this case the ops in question are grouped rather not on the basis of their applicability to sparsity, but on the basis of them being "TF ops" and "weighted", which in turn is more closely related to their appearance in custom layers than to sparsity itself.

Reply:
It is commendable, but the correction does not solve any of the known issues and the change is not consistent. This list is used only in the magnitude algorithm. Please, revert the change.

Reply:
The list is also used in `TFModelConverter._collect_custom_layer_infos`.